src/EntropyComputation.cc

```diff
@@ -0,0 +1,107 @@
+
+#include <mcptam/EntropyComputation.h>
+
+
+//this function evaluates a point's entropy reduction
+double EvaluatePoint(Tracker* tracker, MapPoint& point, KeyFrame& tracker_kf, double priorPointCovariance, int pointLevel, double& prevEntropy)
```

Comment:
Instead of one big function with 7 steps, do you want to separate out the Jacobian of a point calc, the covariance from a Jacobian calc? Would you potentially use those functions elsewhere?